cmd/kubeadm/app/componentconfigs/kubelet_test.go

```diff
@@ -87,6 +88,7 @@ func TestKubeletDefault(t *testing.T) {
 					HealthzPort:        utilpointer.Int32Ptr(constants.KubeletHealthzPort),
 					RotateCertificates: kubeletRotateCertificates,
 					ResolverConfig:     resolverConfig,
+					CgroupDriver:       kubeadmutil.CgroupDriverSystemd,
```

Comment:
you can add constant in constants.go `CgroupDriverSystemd = "systemd"` and use that here instead.
we should remove kubeadmutil.CgroupDriverSystemd

Reply:
Can we just move CgroupDriverSystemd to the right place?
Updated.

Reply:
my idea was to just create a new constant CgroupDriverSystemd in constants (like a copy). so that we don't bind more files to constant in the utils/cgroupdriver file.

i will update in https://github.com/kubernetes/kubernetes/pull/99647